packages/cosmic-swingset/lib/ag-solo/fake-chain.js

```diff
@@ -157,5 +157,8 @@ export async function connectToFakeChain(basedir, GCI, delay, inbound) {
 
   // Start the first pretend block.
   nextBlockTimeout = setTimeout(simulateBlock, maximumDelay);
-  return makeBatchedDeliver(deliver);
+
+  // Only use 100ms batches for no specified inter-block delay.
+  // This makes for more deliveries but less overall waiting time.
+  return makeBatchedDeliver(deliver, delay ? delay * 1000 : 100);
```

Comment:
A nitpick, but can we define delay in a line above so that this call is simplified?